app/Http/Transformers/PostTransformer.php

```diff
@@ -34,9 +35,12 @@ public function transform(Post $post)
             'id' => $post->id,
             'signup_id' => $post->signup_id,
             'northstar_id' => $post->northstar_id,
+            // Add cache-busting query string to urls to make sure we get the
+            // most recent version of the image.
+            // @NOTE - Remove if we get rid of rotation.
             'media' => [
-                'url' => $post->getMediaUrl(),
-                'original_image_url' => $post->url,
+                'url' => $post->getMediaUrl() . '?time='. Carbon::now()->timestamp,
```

Comment:
Can we base this off the post's `updated_at` timestamp, and then `->touch()` it when we rotate a photo?

Reply:
i'd rather not because I think that would create another event record for an update made to the post, which I guess it is, but i'm not sure that is information we need to record

Reply:
Hmm, that's fair. One downside of using the current timestamp here is that people will never get cached resources, so they'll have to re-download all photos every time they load the page. But maybe that's an okay trade-off for now if we're thinking of removing this functionality once we ship Glide image URLs?

Reply:
Yea I was thinking that this would probably relatively temporary. Ultimately, I would like us to re-think some of the event behavior in general so we can have more control of when we fire things off or not. So an update doesn't always trigger a record in the events table. For example, because we touch a signup every time a post is updated, we run into issues with getting event records on signups when nothing has changed, and it is hard to tell what has happened.